_includes/domain-search.html

```diff
@@ -28,7 +28,7 @@
                 <div class="usa-alert usa-alert--success usa-alert--slim" role="alert">
                     <div class="usa-alert__body">
                         <p class="usa-alert__text">
-                        <b>${requested_domain_display_string}</b> is available!
+                        <b title=${requested_domain_display_string} aria-label=${requested_domain_display_string}>${truncateDomain(requested_domain_display_string)}</b> is available!
```

Comment:
(Optional/suggestion) It seems like `<b title=${requested_domain_display_string} aria-label=${requested_domain_display_string}>${truncateDomain(requested_domain_display_string)}</b>` is used multiple times, maybe this could be moved into a variable? 

Totally optional though, that's more of a style thing. Some devs generally prefer to keep html intact for clarity, so it just depends on your preference